python/lsst/validate/drp/photerrmodel.py

```diff
@@ -128,13 +128,20 @@ def fitPhotErrModel(mag, mag_err):
           0.039,  # gamma ('')
           24.35]  # m5 (mag)
 
-    fit_params, fit_param_covariance = curve_fit(
-        photErrModel, mag, mag_err, p0=p0)
+    try:
+        fit_params, fit_param_covariance = curve_fit(
+            photErrModel, mag, mag_err, p0=p0)
+        sigmaSys, gamma, m5 = fit_params[:3]
```

Comment:
My thought was that I knew what I needed, and just wanted to take those.

But I realize that implicitly the reason to sometimes do this is for robustness in case `fit_params` has more parameters than that.  But if the number of parameters in `fit_params` changes, then we actually probably do want to bail rather than silently take the wrong parameters.